README.md

```diff
@@ -19,9 +19,8 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 * Location: virtually  
 
 ## Meetings
-* Meetings are held virtually
+* Meetings are held virtually: [Meeting registration / Join](https://zoom-lfx.platform.linuxfoundation.org/meeting/94112812156?password=f238d6af-c959-48d7-a862-abdb3c648e40)
 * Schedule: bi-weekly, Friday, 2 PM CET/CEST (13:00 UTC, 12:00 UTC during European DST). For date/time of next meeting see previous [meeting minutes](https://github.com/camaraproject/QualityOnDemand/tree/main/documentation/MeetingMinutes).
```

Comment:
What about adding the link to the wiki here and not the archive?  Then the wiki can also include a pointer back to the archive.

Reply:
@RandyLevensalor you are right, that's what I actually wanted to do. Done in https://github.com/camaraproject/QualityOnDemand/pull/253/commits/18e34356b93928a9126f9356148d9ac8f3ad16a4